### PR TITLE
Add local development section to integration a smart contract into a ui

### LINKED ISSUE #201: Add Local Development with npm link section
Update Integrating a  Smart Contract in a UI Docs with a Local Development section that includes using npm link. Add a Setup section.

### PR DESCRIPTION
**Pull request by ymekuria**
Description
closes #201 

Adds section on local development using npm link and a setup section.

### CODE REVIEW

File: pages/en/zkapps/how-to-write-a-zkapp-ui.mdx [hunk 1]
```diff
@@ -26,23 +25,32 @@ This page is a work in progress.
...
 ```
 
+#### Local development
+
+The `npm link` command allows to connect your smart contract to your UI during local development without having to publish it npm.
+
+1. **Link your smart contract:** To link your smart contract package, enter `npm link` on the command line at the root of your smart contract project directory.
+2. **Add the smart contract link to your UI project:** To add the smart contract to your UI, enter `npm link your-package-name` on the command line at the root of your UI project directory. `your-package-name` is the same as the `name` property in the`package.json` of the smart contract project.
+3. Import the linked smart contract package into your UI using: `import { YourSmartContract } from ‘your-package-name’;` , where YourSmartContract is the named export that you chose in your smart contract.
+
```
Comment (jasongitmail) [inline pages/en/zkapps/how-to-write-a-zkapp-ui.mdx:45]: @ymekuria Should we finish this section by mentioning that that they need to run `npm run build` in their smart contract in order to make any changes in their smart contract available in their UI project? I assume it is

**Review by jasongitmail — APPROVED** [review]
@ymekuria Looks good! Left one comment that might be important for devs to understand usability wise.

**Review by iregina — APPROVED** [review]
Great job! Don't see any issues.

**Review by jasongitmail — APPROVED** [review]

File: pages/en/zkapps/how-to-write-a-zkapp-ui.mdx (hunk 1 above)
Comment (jasongitmail) [reply pages/en/zkapps/how-to-write-a-zkapp-ui.mdx:45]: I added this.

Comment (ymekuria) [reply pages/en/zkapps/how-to-write-a-zkapp-ui.mdx:45]: That's a good idea @jasongitmail . I'll add it.

Comment (ymekuria) [reply pages/en/zkapps/how-to-write-a-zkapp-ui.mdx:45]: Looks great!

**Pull request merged**